<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeshProcess</class>
 <widget class="QDialog" name="MeshProcess">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>550</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>204</red>
        <green>232</green>
        <blue>207</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>160</red>
        <green>160</green>
        <blue>160</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>204</red>
        <green>232</green>
        <blue>207</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>160</red>
        <green>160</green>
        <blue>160</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>160</red>
        <green>160</green>
        <blue>160</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>160</red>
        <green>160</green>
        <blue>160</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MeshProcess</string>
  </property>
  <widget class="QComboBox" name="cbMeshProcess">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Laplace smoothing</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>选择处理方式：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>选择平滑次数：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leSmoothCount">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStartProcess">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>161</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="anatomyshow.qrc">
     <normaloff>:/MyImages/images/starthandle.png</normaloff>:/MyImages/images/starthandle.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>160</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpenSourceMesh">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>131</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>打开原始Mesh</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSaveMesh">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>480</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>保存mesh</string>
   </property>
   <property name="iconSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNextStep">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>480</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>下一步</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>220</width>
     <height>380</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>5</number>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <widget class="QVTKWidget" name="vtkSourceMesh">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>13</y>
      <width>200</width>
      <height>360</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>220</width>
     <height>380</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>4</red>
         <green>4</green>
         <blue>4</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>4</red>
         <green>4</green>
         <blue>4</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>4</red>
         <green>4</green>
         <blue>4</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>5</number>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <widget class="QVTKWidget" name="vtkDestMesh">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>13</y>
      <width>200</width>
      <height>360</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnPrevStep">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>480</y>
     <width>321</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>还有部位需要分割？不要紧，返回上一步！</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="anatomyshow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnOpenSourceMesh</sender>
   <signal>clicked()</signal>
   <receiver>MeshProcess</receiver>
   <slot>OpenSourceMesh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnStartProcess</sender>
   <signal>clicked()</signal>
   <receiver>MeshProcess</receiver>
   <slot>StartMeshProcess()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSaveMesh</sender>
   <signal>clicked()</signal>
   <receiver>MeshProcess</receiver>
   <slot>SaveMesh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>660</x>
     <y>510</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnNextStep</sender>
   <signal>clicked()</signal>
   <receiver>MeshProcess</receiver>
   <slot>NextStep()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>510</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPrevStep</sender>
   <signal>clicked()</signal>
   <receiver>MeshProcess</receiver>
   <slot>PrevStep()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>510</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>StartMeshProcess()</slot>
  <slot>OpenSourceMesh()</slot>
  <slot>SaveMesh()</slot>
  <slot>NextStep()</slot>
  <slot>PrevStep()</slot>
 </slots>
</ui>
